File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/examples/simple/ExpZPowMinusOneHalf.java
package net.fwitz.math.main.binary.complex.functions.examples.simple;

import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class ExpZPowMinusOneHalf {
    public static void main(String[] args) {
        new ComplexFunctionPlot("exp(1 / z^(1/2))")
                .computeFn(z -> z.pow(0.5).inverse().exp())
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/examples/simple/ExpZPowMinusOneHalf.kt
package net.fwitz.math.main.binary.complex.functions.examples.simple

import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot

object ExpZPowMinusOneHalf {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("exp(1 / z^(1/2))")
        .computeFn { z -> z.sqrt.inverse.exp }
        .render()
}